editor/js/ui/editor.js

```diff
@@ -413,10 +422,10 @@ RED.editor = (function() {
                         }
                     } else {
                         console.log("Unknown type:", definition.defaults[d].type);
-                        preparePropertyEditor(node,d,prefix);
+                        preparePropertyEditor(node,d,prefix,definition.defaults);
```

Comment:
Ah you are right. The comment you had added to `preparePropertyEditor` for that additional parameter says:

    @param definition - the definition of the field

which implied to me it was an individual property definition. But looking at the code I see it is the full set of properties definitions.

I think I'll end up changing that to be the individual property def - no need to pass in all of them. But I'll do that after this gets merged.